src/homework6/modelfromlife.py

```diff
@@ -0,0 +1,275 @@
+import random
+
+"""Создайте  модель из жизни. Это может быть бронирование комнаты в
+отеле, покупка билета в транспортной компании, или простая РПГ.
+Создайте несколько объектов классов, которые описывают ситуацию
+Объекты должны содержать как атрибуты так и методы класса для
+симуляции различных действий. Программа должна инстанцировать
+объекты и эмулировать какую-либо ситуацию - вызывать методы,
+взаимодействие объектов и т.д.
+
+"""
+
+
+class Casino(object):
+
+    def __init__(self, name):
+        self.list_guests = list_guests[:]
+        self.name = name
+        self.roulette_table = [val for val in range(37)]
+        self.blackjack_deck = [1, 2, 3, 4, 5, 6, 7,
+                               8, 9, 10, 10, 10, 10
+                               ]
+        self.one_hand_bandit_ = [1, 2, 3, 4, 5, 6, 7]
+        self.start_capital = 10000
+        self.capital = 10000
+        self.roulette_players = []
+        self.blackjack_players = []
+        self.bar_visitors = []
+
+    def cycle(self, times):
```

Comment:
it should be moved out of casino class at all

Reply:
moved out